scripts/install-deps.sh

```diff
@@ -190,10 +190,17 @@ do_install_aduc_packages() {
     OS=$(lsb_release --short --id)
     if [[ $OS == "Debian" && $VER == "9" ]]; then
         $SUDO apt-get install --yes gcc-6 g++-6 || return
-    elif [[ $OS == "Debian" && $VER == "11" ]]; then
-        $SUDO apt-get install --yes gcc-10 g++-10 || return
-    elif [[ $OS == "Ubuntu" && $VER == "22.04" ]]; then
+    elif [[ ($OS == "Debian" && $VER == "11") || (\
+        $OS == "Ubuntu" && $VER == "20.04") || (\
+        $OS == "Ubuntu" && $VER == "22.04") ]] \
+            ; then
         $SUDO apt-get install --yes gcc-10 g++-10 || return
+
+        # used for dependencies like catch2 that will find system default
```

Comment:
I'm not thrilled about us modifying user environment for our needs.  Is there no way for us to build catch2 and pass along "CC=" and "CXX=" on the cmake commandline (or however we build catch2)?

Reply:
Good suggestion. Yes, have pushed a commit that does this now.